Source/Core/DolphinWX/GameListCtrl.cpp

```diff
@@ -1075,7 +1075,8 @@ void CGameListCtrl::CompressSelection(bool _compress)
 		for (u32 i = 0; i < m_numberItem; i++)
 		{
 			const GameListItem* iso = GetSelectedISO();
-			if (iso->GetPlatform() == DiscIO::IVolume::WII_WAD || iso->GetFileName().rfind(".wbfs") != std::string::npos)
+			if (iso->GetPlatform() != DiscIO::IVolume::GAMECUBE_DISC || iso->GetPlatform() != DiscIO::IVolume::WII_DISC ||
+			    iso->GetBlobType() != DiscIO::BlobType::PLAIN || iso->GetBlobType() != DiscIO::BlobType::GCZ)
```

Comment:
Either my brain is completely wrong today, or this code is.

Since iso->GetPlatform() can't be both GAMECUBE_DISC and WII_DISC at the same time, one of the first two expressions will **Always** be true and we will always continue.

The same is true for the second two expressions, but since one of the first two expressions will always be true, there is no need to even bother evaluating them.

Did you want `&&` instead of `||`?